src/core/model_repository_manager.cc

```diff
@@ -256,12 +256,13 @@ class ModelRepositoryManager::BackendLifeCycle {
   ~BackendLifeCycle() { map_.clear(); }
 
   // Start loading model backends with specified versions asynchronously.
-  // If 'force_unload', all versions that are being served will
-  // be unloaded before loading the specified versions.
+  // If 'defer_unload' is false, all versions that are being served will
+  // be unloaded before loading the specified versions. Otherwise, the versions
+  // not specified in the load will be unloaded after the load is finished.
   Status AsyncLoad(
       const std::string& repository_path, const std::string& model_name,
       const std::set<int64_t>& versions,
-      const inference::ModelConfig& model_config, bool force_unload = true,
+      const inference::ModelConfig& model_config, bool defer_unload = false,
```

Comment:
So when we call AsyncLoad we explicitly set defer_unload to true?

Reply:
Yes